src/data.js

```diff
@@ -806,44 +806,31 @@ export const disciplines = {
 export const vitals = [
   {
     id: "Health",
-    style: "healthPt",
     scale: 15,
-    depends: ['Stamina'],
-    value: 3,
     defaultValue: 3,
     description:"Your Health; usually equals stamina + 3",
   },
   {
     id: "Willpower",
-    style: "willPt",
     scale: 15,
-    depends: ['Composure', 'Resolve'],
-    value: 0,
     defaultValue: 0,
     description:"Your Willpower; usually equals composure + resolve",
   },
   {
     id: "Humanity",
-    style: "humanityPt",
     scale: 10,
-    value: 7,
-    defaultValue: 7, 
+    defaultValue: 0, 
     description:"Your Humanity, a measure of resistance against the beast",
   },
   {
     id: "Hunger",
-    style: "hungerPt",
     scale: 5,
-    value: 1,
-    defaultValue: 1,
+    defaultValue: 0,
```

Comment:
hunger zacina na 1

Reply:
myslel som si, ze default alebo initial value mas inac znacene csskom

Reply:
ze rozdiel je ten, ze sa to neda odobrat, ale mozno to chapem zle

Reply:
hej bolo to tak... hunger by bol vyijmka

Reply:
proste by sa v evente povedalo ze pre kazdy ten stat sa to chova trocha inac, blood potency to hned zahodi, pre heath, will to urobi nieco, pre humanity, hunger nieco ine

Reply:
vo vysledku po tych zmenach, co som robil sa pocita, ze konecna hodnota je initial + computed
inital navyse dostanu inu css triedu. Neviem ci v css je pre to zvlast trieda a ma to vypadat inac, ale pre istotu som to zmenil, aby css zostalo rovnake pre vsetky. Samozrejme ak to chces zmenit, tak to zmen